src/pages/centro-historico.tsx

```
@@ -0,0 +1,801 @@
+                    <p className="text-sm text-gray-600">{t('centroHistorico.practical.safetyInfoText')}</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            {/* Tips Card */}
+            <div className="bg-gradient-to-br from-amber-600 to-amber-700 rounded-xl p-8 text-white">
+              <h3 className="text-xl font-bold mb-6">{t('centroHistorico.tips.title')}</h3>
+              <ul className="space-y-4">
+                <li className="flex items-start space-x-3">
+                  <svg className="w-5 h-5 text-amber-200 mt-1" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)